src/internal/utils/validators.ts

```diff
@@ -80,6 +80,20 @@ export function validateListSliceStartEnd(
   }
 }
 
+export function validateTopicName(name: string) {
+  if (isEmpty(name)) {
+    throw new InvalidArgumentError('topic name must not be empty');
+  }
+}
+
+export function validateTopicValue(value: string | Uint8Array) {
+  if (typeof value === 'string') {
```

Comment:
I modeled this after the cache validation already in place. I also noticed that there is no validation on `Uint8Array` in the code or in tests, so that is something to look in to. If someone passes an object or an integer then it would throw an exception from the gRPC guts. I can add a branch to check for that. The client code is vulnerable to this as well.

Reply:
I'm with Pete though, why do we want to disallow sending empty messages?

Reply:
Reverted in https://github.com/momentohq/client-sdk-nodejs/pull/356/commits/eee047d1d07d0487b64c8d7a5ea34a81a1bdab5a